distributed/scheduler.py

```diff
@@ -5411,15 +5415,15 @@ def handle_missing_data(self, key=None, errant_worker=None, **kwargs):
             else:
                 self.transitions({key: "forgotten"})
 
-    def release_worker_data(self, comm=None, keys=None, worker=None):
+    def release_worker_data(self, comm=None, key=None, worker=None):
         parent: SchedulerState = cast(SchedulerState, self)
+        if worker not in parent._workers_dv:
+            return
         ws: WorkerState = parent._workers_dv[worker]
```

Comment:
nitpick: use get for both ws and ts
```suggestion
        ws: WorkerState = parent._workers_dv.get(worker)
```